terraform/stanleyyv-dev/us-west-2/ec2.tf

```diff
@@ -0,0 +1,18 @@
+# AWS EC2 instance 
+# instance is in region us-west-2
+resource "aws_instance" "stanley-amazon-linux-1" {
+  ami = "ami-0c2ab3b8efb09f272"
+  instance_type = "t2.micro"
+  
+  tags = {
+    Name = "EC2InstanceByTerraform"
```

Comment:
you might want to keep the tagging naming convention consistent with the other resources. 

* `Name` : <env-region-suffix>
* `terraform` : <true/false>